Minor client changes.

NiteshKant opened the pull request:
1) Modified RxClientImpl.connect() to do a take(1) on the underlying source so that in no case it will emit two onNext().
2) Modified RequestProcessingOperator to use SubscriptionList instead of CompositeSubscription. Instead of creating a new subscription on every onNext(), it is done once and used by the subscriber.
3) Provided an override to remove all connection pooling for a client.

Commit:
Minor client changes.

1) Modified RxClientImpl.connect() to do a take(1) on the underlying source so that in no case it will emit two onNext().
2) Modified RequestProcessingOperator to use SubscriptionList instead of CompositeSubscription. Instead of creating a new subscription on every onNext(), it is done once and used by the subscriber.
3) Provided an override to remove all connection pooling for a client.

Commit:
Updated Changes.md with maven artifact links.

cloudbees-pull-request-builder commented:
[RxNetty-pull-requests #58](https://netflixoss.ci.cloudbees.com/job/RxNetty-pull-requests/58/) ABORTED

NiteshKant commented:
I am looking at why all pull request builds are getting aborted.

Commit:
Adding release 0.3.6 to Changes.md

cloudbees-pull-request-builder commented:
[RxNetty-pull-requests #61](https://netflixoss.ci.cloudbees.com/job/RxNetty-pull-requests/61/) ABORTED

Commit:
Removing infinite waits from tests.

Many test cases were doing an infinite wait for the mock server to shutdown. This was more required when the servers were not using ephemeral ports.
Introduced a method of time wait on RxServer and using that method in test.

Commit:
Removing method deprecated in rxjava-0.0.19.

- toBlockingObservable() is deprecated in favor of toBlocking()
- SubscriptionList class is moved to rxjava util pkg.

cloudbees-pull-request-builder commented:
[RxNetty-pull-requests #63](https://netflixoss.ci.cloudbees.com/job/RxNetty-pull-requests/63/) SUCCESS
This pull request looks good

NiteshKant commented:
There were some infinite waits for server shutdown in some cases which were causing the build in cloudbee to hang, strangely so only for pull request builds. I have replaced these waits with timed waits.
